src/Entities/ChainStuff/ChainedKevin.cs

```diff
@@ -0,0 +1,255 @@
+﻿using Celeste.Mod.Entities;
+using Microsoft.Xna.Framework;
+using Mono.Cecil.Cil;
+using Monocle;
+using MonoMod.Cil;
+using MonoMod.Utils;
+using System;
+using System.Collections.Generic;
+using static Celeste.MoveBlock;
```

Comment:
To only import a type as a specific name, which is less likely to produce name conflicts, use
```suggestion
using Directions = Celeste.MoveBlock.Directions;
```